### PR TITLE
Copter: Add note about WoW to land-mode.rst

### PR DESCRIPTION
**Pull request by TunaLobster**
This is for https://github.com/ArduPilot/ardupilot/pull/16506

### CODE REVIEW

File: copter/source/docs/land-mode.rst [hunk 1]
```diff
@@ -31,6 +31,12 @@ features:
     second.  It does not use the altitude to decide whether to shut off the
     motors except that the copter must also be below 10m above the home
     altitude.
+    
+.. note::
+    
+    Using a Weight on Wheels (WoW) switch will increase the decent rate and
```
Comment (Hwurzburg) [inline copter/source/docs/land-mode.rst:37]: spelling  descent

File: copter/source/docs/land-mode.rst [hunk 2]
```diff
@@ -31,6 +31,12 @@ features:
...
+    accelerometer ranges that are acceptable for landing detection. This
+    feature is enabled anytime that LGR_WOW_PIN is not disabled.
```
Comment (Hwurzburg) [inline copter/source/docs/land-mode.rst:39]: anytime the...

Comment (Hwurzburg) [reply copter/source/docs/land-mode.rst:39]: are there any other relevant parameters needing setup to make this work correctly?

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: copter/source/docs/land-mode.rst (hunk 1 above)
Comment (TunaLobster) [reply copter/source/docs/land-mode.rst]: That's what I get for trying to work without a spell checker.

File: copter/source/docs/land-mode.rst (hunk 2 above)
Comment (TunaLobster) [reply copter/source/docs/land-mode.rst]: LGR_WOW_POL for the polarity of the switch, but this feature is not dependent on that parameter.

Comment (Hwurzburg) [reply copter/source/docs/land-mode.rst]: @TunaLobster Charlie, needs merge conflicts resolved before it can be merged..thanks

**Review by Hwurzburg — APPROVED** [review]
fixed the conflict on rebase, pushed and merging...thanks

**Pull request merged**